pkg/runtime/conversion_generator.go

```diff
@@ -302,11 +302,15 @@ func (g *conversionGenerator) generateConversionsForSlice(inType, outType reflec
 }
 
 func (g *conversionGenerator) generateConversionsForStruct(inType, outType reflect.Type) error {
+	errs := []string{}
```

Comment:
Yes,  the early return from this method seemed to prevent the generation of public convert methods for other fields in the type unnecessarily.